docs/eos-learn/accounts-and-permissions.md

```diff
@@ -0,0 +1,62 @@
+---
+id: accounts-and-permissions
+title: Accounts and Permissions
+sidebar_label: Accounts and Permissions
+---
+
+## Accounts
+
+An **account** on EOSIO is a legible name for humans that get stored on the blockchain. An account can belong to an individual or group of individuals depending on the permissions configuration. An account is required in order to execute any transaction on the blockchain.
+
+The account name must comply with certain requirements: characters from `a` to `z` in lower case, numbers from `1` to `5` and have 12 characters length. E. g. `eosioaccount12`
```

Comment:
@JeanVegaD  accounts need to be 12 characters in length.   `eosioaccount` works great `eosioaccount12` is too long!